packages/firebase_dynamic_links/lib/src/firebase_dynamic_links.dart

```diff
@@ -37,6 +37,16 @@ class FirebaseDynamicLinks {
     return getPendingDynamicLinkDataFromMap(linkData);
   }
 
+  Future<Uri> getDynamicLink(Uri url) async {
```

Comment:
Is there a reason why this returns a Uri vs `PendingDynamicLinkData`?

Reply:
Changed to PendingDynamicLinkData